### PR TITLE
Assignment3

### PR DESCRIPTION
**Pull request by Denrig**
Heroku Page: http://wolfpack-restaurant.herokuapp.com/

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
-  root 'application#home'
+  root 'products#index'
+
+  resources :products
```
Comment (VictorMotogna) [inline config/routes.rb:5]: you can do here:
```ruby
resources :products, only: %i[index new create]
```

File: app/assets/stylesheets/products.scss [hunk 2]
```diff
@@ -0,0 +1,217 @@
+//SORTING BAR
+$SORTING_BACKGROUND_COLOR: #a0999a88;
```
Comment (florinionce) [inline app/assets/stylesheets/products.scss:2]: usually, the sass variables are defined with lower case -> https://sass-lang.com/documentation/variables

File: app/assets/stylesheets/products.scss [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      width: 500px !important;
+      border: 1px solid orange;
+      background-color: #f0f0f0;
+      h2 {
+        margin: 0;
+        background-color: orange;
+        height: 3vh;
+      }
+    }
+    .input-field,
```
Comment (florinionce) [inline app/assets/stylesheets/products.scss:188]: missing space 👀

File: app/assets/stylesheets/scaffolds.scss [hunk 4]
```diff
@@ -1,52 +1,59 @@
...
+  display: table;
+}
 
 #error_explanation {
   width: 450px;
   border: 2px solid red;
   padding: 7px 7px 0;
   margin-bottom: 20px;
-  background-color: #f0f0f0; }
+  background-color: #f0f0f0;
```
Comment (florinionce) [inline app/assets/stylesheets/scaffolds.scss:55]: shouldn't this be a variable?

File: app/models/product.rb [hunk 5]
```diff
@@ -0,0 +1,10 @@
+class Product < ApplicationRecord
+  has_one_attached :image
+
+  validates :title, :price, :category, :image, presence: true
+  validates :title, format: { with: /\A[a-zA-Z ]+\z/,
+                              message: 'Title must only contain letters' },
+                    length: { minimum: 2, maximum: 20 }
+  validates :price, numericality: { only_integer: true }
+  validates :category, length: { maximum: 20 }
```
Comment (florinionce) [inline app/models/product.rb:9]: nice job with the validations 👍

File: app/views/products/_form.html.haml [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    = form.label :category
+    = form.text_field :category
+  .input-field
+    = form.label :price
+    = form.number_field :price
+  .input-field
+    = form.label :image
+    = form.file_field :image
+  .actions
+    =form.submit
```
Comment (florinionce) [inline app/views/products/_form.html.haml:22]: missing space, also you should trim whitespaces :D

**Pull request merged**